README.md

```diff
@@ -54,8 +54,8 @@ This should start the containers and allow you to setup the database.
 
 To initialize the database (create user, tables, etc.) run these commands:
 
-    $ docker-compose run web python manage.py init_db --skip-create
-    $ docker-compose run web python manage.py init_msb_db 
+    $ docker-compose run web python manage.py init_db 
+    $ docker-compose run web python manage.py init_msb_db --create-db
```

Comment:
yeah, ruaok and I discussed this a little bit. We should be able to have a docker image for messybrainz which sets up the database, and then (somehow) install the messybrainz database code into the LB env so that it can connect to that database directly